features/step_definitions/scheduler.rb

```diff
@@ -0,0 +1,19 @@
+Given /^the #{QUOTED} scheduler CRD is restored after scenario$/ do |name|
+  ensure_admin_tagged
+  org_scheduler = {}
+  @result = admin.cli_exec(:get, resource: 'scheduler', resource_name: name, o: 'yaml')
+  if @result[:success]
+    org_scheduler['spec'] = @result[:parsed]['spec']
+    logger.info "scheduler restore tear_down registered:\n#{org_scheduler}"
+  else
+    raise "Could not get scheduler: #{name}"
+  end
+  patch_json = org_scheduler.to_json
+  _admin = admin
+  teardown_add {
+    opts = {resource: 'scheduler', resource_name: name, p: patch_json, type: 'merge' }
+    @result = _admin.cli_exec(:patch, **opts)
+    raise "Cannot restore scheduler: #{name}" unless @result[:success]
+    sleep 60
```

Comment:
@zhouying7780, can you please elaborate the reason for the 60 seconds hard-coded sleep?

Reply:
Agree, for any magic number, always encouraged to comment the reason

Reply:
@pruan-rht @xingxingxia  I just copy the method from features/step_definitions/oauth.rb, it         has defined the 60 seconds sleep.

Reply:
deleted , and retest , no issue.